src/test/java/ru/training/at/hw2/ex1/FirstExerciseTest.java

```diff
@@ -0,0 +1,79 @@
+package ru.training.at.hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import ru.training.at.hw2.TestsInitialization;
+
+public class FirstExerciseTest extends TestsInitialization {
+
+    @Test
+    public void exerciseTest() {
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
```

Comment:
You have that point in description: Block (method) should start with comment line (ex. “// 2. Assert Browser title”) with step description which has been taken from this test case file.

So let's separate each step from exercise with name from description and one additional empty line. It will be more readable.